lib/src/form_builder_validators.dart

```diff
@@ -206,9 +206,10 @@ class FormBuilderValidators {
   static FormFieldValidator integer(
     BuildContext context, {
     String errorText = 'Value must be an integer.',
+    int radix,
   }) {
     return (valueCandidate) {
-      if (int.tryParse(valueCandidate ?? '') == null) {
+      if (int.tryParse(valueCandidate ?? '', radix: radix) == null) {
```

Comment:
I like the `radix` add, but the `?? ''` is still a problem.

Reply:
Done.